src/test_snmp.cpp

```diff
@@ -406,3 +407,152 @@ TEST_F(SNMPTest, SingleCountByNodeTypeTable)
   cwtest_reset_time();
   delete tbl;
 }
+
+TEST_F(SNMPTest, SuccessFailCountByRequestTypeTable)
+{
+  cwtest_completely_control_time();
+
+  // Create a table
+  SNMP::SuccessFailCountByRequestTypeTable* tbl = SNMP::SuccessFailCountByRequestTypeTable::create("success_fail_by_request", test_oid);
+  FILE* fd;
+  char buf[1024];
+
+  // To start with, all values should be 0.
+  // Shell out to snmpwalk to find previous 5 second period attempts.
```

Comment:
Worth noting in the comment what the SIP request type we're querying is, as well as the time period.